### ISSUE TITLE
DASH-HLS Interop and MPEG-DASH CMAF Profile

### ISSUE THREAD

**Issue by johnsim** [cta-wave/content-specification-task-force#45]
An upcoming Draft Amendment to DASH will include a DASH CMAF Profile. Is there feature parity between DASH and HLS for this DASH CMAF Profile? Can an interop profile be defined based on the DASH CMAF Profile?

**Comment by johnsim** [comment]
From DASH CMAF DAM: “In order to distribute CMAF content in DASH, this profile defines a normative mapping of CMAF structures to DASH structures. Using this mapping enables content to conform to both CMAF constraints and DASH requirements.” There are 38 shall statements in the section defining the profile. Can the same normative mapping be created for HLS?

**Comment by johndeu** [comment]
@johnsim  - are we including Low latency HLS/CMAF and DASH interop in this issue topic?

**Comment by johnsim** [comment]
yes - the work is moving forward in WAVE in a new ad hoc group lead by Zack Cava (Hulu). It is use case driven, and one of the interop use cases is low latency streaming - as discussed this morning in the WAVE virtual F2F meeting - aka a longer than usual video conference call.
